### PR TITLE
pydeck: Enforce pydeck notebook deck.gl semver on sdist

### PR DESCRIPTION
**Pull request by ajduberstein**
For #4103 

#### Change List
-

### CODE REVIEW

**Comment by Pessimistress** [comment]
If I understand this correctly, it will block the ability to publish beta versions of pydeck that depends on a beta version of deck.gl.

**Comment by ajduberstein** [comment]
> If I understand this correctly, it will block the ability to publish beta versions of pydeck that depends on a beta version of deck.gl.

There's nothing currently enforcing that a particular prerelease of pydeck depends on a prerelease of deck.gl, thought I can add that check. This just simply makes sure that a maintainer is intentional about the deck.gl release paired with pydeck.

File: bindings/pydeck/pydeck/frontend_semver.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+"""Current associated version of NPM modules deck.gl and @deck.gl/jupyter-widget"""
```
Comment (ajduberstein) [inline bindings/pydeck/pydeck/frontend_semver.py]: todo: read from lerna.json instead

Comment (ajduberstein) [reply bindings/pydeck/pydeck/frontend_semver.py]: also check that the correct tag matches the current branch (beta/alpha from master only)

File: bindings/pydeck/pydeck/frontend_semver.py [hunk 2]
```diff
@@ -0,0 +1 @@
+DECKGL_SEMVER = "^8.0.0"
```
Comment (Pessimistress) [inline bindings/pydeck/pydeck/frontend_semver.py]: Add this to .gitignore if it's automatically generated

**Pull request merged**